frameworks/PHP/php/php-caddy.dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
+RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php > /dev/null && \
+    apt-get update -yqq > /dev/null && apt-get upgrade -yqq 
+
+RUN apt-get install -yqq nginx git unzip curl \
+    php8.1 php8.1-common php8.1-cli php8.1-fpm php8.1-mysql > /dev/null
+
+COPY deploy/conf/* /etc/php/8.1/fpm/
+
+# Install Caddyserver
+RUN apt-get install -y debian-keyring debian-archive-keyring apt-transport-https > /dev/null \
+    && curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg \
+    && curl -sf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | tee /etc/apt/sources.list.d/caddy-stable.list \
+    && apt-get update > /dev/null && apt-get install caddy > /dev/null
+
+ADD ./ /php
+WORKDIR /php
+
+RUN if [ $(nproc) = 2 ]; then sed -i "s|pm.max_children = 1024|pm.max_children = 512|g" /etc/php/8.1/fpm/php-fpm.conf ; fi;
+
+RUN chmod -R 777 /php
+
+EXPOSE 8080
+
+CMD service php8.1-fpm start && \
+    caddy run --config deploy/caddy/Caddyfile
```

Comment:
This isn't ideal, because `SIGTERM` won't properly shut things down.

Reply:
I'm just coming here from being a Caddy maintainer. If this is typical here, then :man_shrugging: but it definitely doesn't follow Docker best practices.

Reply:
In reality the toolset, stop the entire docker container when finish.

Reply:
I ask you again, How would you do it ?
I'm not a member from TechEmpower.

Reply:
>  I'm not a member from TechEmpower.

I'm not either.

Reply:
Thank you